cohortextractor/tpp_backend.py

```diff
@@ -234,8 +235,9 @@ def get_queries(self, covariate_definitions):
                 )
             else:
                 date_format_args = pop_keys_from_dict(query_args, ["date_format"])
-                cols, sql = self.get_query(name, query_type, query_args)
+                cols, sql, setup_sql = self.get_query(name, query_type, query_args)
                 table_queries[name] = f"SELECT * INTO #{name} FROM ({sql}) t"
+                table_setup_queries[name] = setup_sql
```

Comment:
Looks like `setup_sql` will  be iterated over -- can that be reflected in its name?